beaver/worker/tail.py

```diff
@@ -207,6 +207,9 @@ def _ensure_file_is_good(self, current_time):
             if err.errno == errno.ENOENT:
                 self._log_info('file removed')
                 self.close()
+                return
+            else:
```

Comment:
@josegonzalez trying to return only if errorcode is for file not found, else raise the exception since I'd imagine other errors stat-ing the file would be unexpected.